conf/test.config

```diff
@@ -16,7 +16,7 @@ params {
     config_profile_description = 'Minimal test dataset to check pipeline function'
 
     // Limit resources so that this can run on GitHub Actions
-    max_cpus   = 2
+    max_cpus   = 4
```

Comment:
4 works on GHA?

Reply:
don't think, so I wanted to speed up my tests locally :D